File: Tutorials/OpenGLLayerExample/netcore/ImagedButton.cs
using System.Drawing;
using SpaceVIL;
using SpaceVIL.Core;

namespace OpenGLLayerExample
{
    public class ImagedButton : ButtonCore
    {
        ImageItem _image = null;

        public ImagedButton(Bitmap image, float imageRotationAngle)
        {
            SetBackground(0xFF, 0xB5, 0x6F);
            SetSizePolicy(SizePolicy.Fixed, SizePolicy.Fixed);
            SetSize(30, 30);
            SetAlignment(ItemAlignment.Right, ItemAlignment.Bottom);
            SetPadding(5, 5, 5, 5);
            SetShadow(5, 0, 2, Color.Black);
            IsFocusable = false;

            _image = new ImageItem(image, false);
            _image.SetRotationAngle(imageRotationAngle);
            _image.SetColorOverlay(Color.White);
        }

        public override void InitElements()
        {
            base.InitElements();
            AddItem(_image);
        }
    }
}

File: Tutorials/OpenGLLayerExample/java/src/main/java/OpenGLLayerExample/ImagedButton.java
package OpenGLLayerExample;

import java.awt.image.BufferedImage;
import java.awt.Color;

import com.spvessel.spacevil.ButtonCore;
import com.spvessel.spacevil.ImageItem;
import com.spvessel.spacevil.Flags.ItemAlignment;
import com.spvessel.spacevil.Flags.SizePolicy;

public class ImagedButton extends ButtonCore {
    ImageItem _image = null;

    public ImagedButton(BufferedImage image, float imageRotationAngle) {
        setBackground(0xFF, 0xB5, 0x6F);
        setSizePolicy(SizePolicy.FIXED, SizePolicy.FIXED);
        setSize(30, 30);
        setAlignment(ItemAlignment.RIGHT, ItemAlignment.BOTTOM);
        setPadding(5, 5, 5, 5);
        setShadow(5, 0, 2, Color.BLACK);
        isFocusable = false;

        _image = new ImageItem(image, false);
        _image.setRotationAngle(imageRotationAngle);
        _image.setColorOverlay(Color.WHITE);
    }

    @Override
    public void initElements() {
        super.initElements();
        addItem(_image);
    }
}
